src/contrib/serializers/Akka.Serialization.TestKit/AkkaSerializationSpec.cs

```diff
@@ -145,7 +145,7 @@ public void CanSerializeProps()
         [Fact]
         public void CanSerializeDeploy()
         {
-            var message = new Deploy(RouterConfig.NoRouter).WithMailbox("abc");
+            var message = new Deploy(NoRouter.Instance).WithMailbox("abc");
```

Comment:
Came up with a compromise - added `RouterConfig.NoRouter` back, but it's implemented like this:

```csharp
public static RouterConfig NoRouter => NoRouter.Instance;
```

I also marked that API as obsolete and told users to start using `NoRouter.Instance` instead.